libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTracker.java

```diff
@@ -139,6 +139,7 @@
         EDITOR_ADDED_PHOTO_VIA_WP_MEDIA_LIBRARY,
         EDITOR_ADDED_VIDEO_VIA_WP_MEDIA_LIBRARY,
         EDITOR_ADDED_PHOTO_VIA_STOCK_MEDIA_LIBRARY,
+        EDITOR_ADDED_FILE_VIA_LIBRARY,
```

Comment:
@marecar3 I am thinking there should be a case for this in `getEventNameForStat` of `AnalyticsTrackerNosara` 🙏

Reply:
Thanks, @jd-alexander, you are right.
I have added the case.